server/request_processor.go

```diff
@@ -53,7 +62,14 @@ func (r *RpcRequest) ProcessRequest() *types.JsonRpcResponse {
 
 	switch {
 	case r.jsonReq.Method == "eth_sendRawTransaction":
+		r.ethSendRawTxEntry = &database.EthSendRawTxEntry{
+			IsWhiteHatBundleCollection: r.isWhitehatBundleCollection,
+			WhiteHatBundleId:           r.whitehatBundleId,
+			Id:                         r.id,
+			RequestId:                  r.requestId,
+		}
 		r.handle_sendRawTransaction()
+		r.db.SaveEthSendRawTxEntry(r.ethSendRawTxEntry)
```

Comment:
do not pass the database in to request, the whole thing should be saved at once by the handler.